sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRegistryClientsTestBase.java

```diff
@@ -105,7 +105,7 @@ ContainerRegistryClientBuilder getContainerRegistryBuilder(HttpClient httpClient
             .credential(credential)
             .audience(audience);
 
-           // builder.httpClient(new NettyAsyncHttpClientBuilder().proxy(new ProxyOptions(ProxyOptions.Type.HTTP, new InetSocketAddress("localhost", 8888))).build());
+            // builder.httpClient(new NettyAsyncHttpClientBuilder().proxy(new ProxyOptions(ProxyOptions.Type.HTTP, new InetSocketAddress("localhost", 8888))).build());
```

Comment:
I like to keep this handy, since it is really helpful while testing\debugging things. Unless you feel strongly about it, I would really like to keep this in the file.

Reply:
We can keep it commented out if it's useful for testing. Another option is to hide it behind an env variable (ACR_PROXY_ENABLED) and set the httpClient only when this env var is true.